docs/data-model.asciidoc

```diff
@@ -378,6 +427,40 @@ include::./data-streams.asciidoc[tag=metrics-data-streams]
 
 See <<apm-data-streams>> to learn more.
 
+[float]
+==== Aggregated metrics: limits and overflows
+
+For all aggregated metrics, namely transaction, service-transaction, service-destination, and service-summary metrics,
+there are limits on the number of unique groups tracked at any given time.
+
+[float]
+===== Limits
+
+For transaction metrics, the limits are 1000 services per GB of APM Server, 5000 transaction
+groups per GB of APM Server, and 500 transaction groups per service per GB of APM Server.
+
+For service-transaction metrics, the limit is 1000 service transaction groups per GB of APM Server.
+
+For service-destination metrics, the limit is a constant of 10000 service destination groups.
+
+For service-summary metrics, the limit is 1000 service summary groups per GB of APM Server.
+
+Note that all of the above limits may change in the future with further improvements.
+
+[float]
+===== Overflows
+
+When a dimension has a high cardinality and exceeds the limit, the metrics will be aggregated
+under a dedicated overflow bucket.
+
+For example, when `transaction.name` has a lot of unique values and reaches the limit
+of unique transaction groups tracked, any transactions with new `transaction.name` will all be aggregated under
+`transaction.name`: `_other`.
```

Comment:
Please add another example where the transaction.name cardinality is modest but the service has a lot of instances.